sbt-plugin/src/sbt-test/gen-java/01-gen-basic-java/build.sbt

```diff
@@ -7,9 +7,7 @@
  * This file is part of the Apache Pekko project, which was derived from Akka.
  */
 
-// Can be removed when we move to 2.12.14
-// https://github.com/akka/akka-grpc/pull/1279
-scalaVersion := "2.12.20"
+scalaVersion := "2.12.21"
```

Comment:
are we sure we still need the explicit scala version here? the comment/issue seems to suggest it was only added because of a specific problem with 2.12.13